src/main/java/uk/gov/hmcts/darts/arm/service/impl/BatchCleanupArmResponseFilesServiceCommon.java

```diff
@@ -48,18 +48,34 @@ public class BatchCleanupArmResponseFilesServiceCommon implements BatchCleanupAr
     protected final CurrentTimeHelper currentTimeHelper;
     protected final ArmResponseFileHelper armResponseFileHelper;
     protected final String manifestFilePrefix;
+    protected final String loggingPrefix;
+
+    public BatchCleanupArmResponseFilesServiceCommon(ExternalObjectDirectoryRepository externalObjectDirectoryRepository,
+                                                     ObjectRecordStatusRepository objectRecordStatusRepository,
+                                                     ExternalLocationTypeRepository externalLocationTypeRepository,
+                                                     ArmDataManagementApi armDataManagementApi,
+                                                     UserIdentity userIdentity,
+                                                     ArmBatchCleanupConfiguration batchCleanupConfiguration,
+                                                     ArmDataManagementConfiguration armDataManagementConfiguration,
+                                                     CurrentTimeHelper currentTimeHelper,
+                                                     ArmResponseFileHelper armResponseFileHelper,
+                                                     String manifestFilePrefix) {
+        this(externalObjectDirectoryRepository, objectRecordStatusRepository, externalLocationTypeRepository, armDataManagementApi, userIdentity,
+             batchCleanupConfiguration, armDataManagementConfiguration, currentTimeHelper, armResponseFileHelper,
+             manifestFilePrefix, manifestFilePrefix);
```

Comment:
If you make the manifest prefix an abstract method and in the parent classes they pass the appropriate prefix you wont need to pass them in the constructor

Reply:
A method feels wrong for this, as we want it to be a constant value which we can not guarantee with a method. I have updated the code to use overloaded constructors to bypass the need to pass in the value twice for methods this

Reply:
@hemantasharma1129  @jackmaloney  what are your views?